session/session.go

```diff
@@ -64,16 +82,77 @@ func (s *Session) Has(key string) bool {
 	return val != nil
 }
 
+func (s *Session) Invalidate() error {
+	s.Flush()
+	return s.Migrate(true)
+}
+
+func (s *Session) Migrate(destroy ...bool) error {
+	shouldDestroy := false
+	if len(destroy) > 0 {
+		shouldDestroy = destroy[0]
+	}
+
+	if shouldDestroy {
+		err := s.driver.Destroy(s.GetID())
+		if err != nil {
+			return err
+		}
+	}
+
+	s.SetID(s.generateSessionID())
+
+	return nil
+}
+
+func (s *Session) Missing(key string) bool {
+	return !s.Exists(key)
+}
+
+func (s *Session) Only(keys []string) map[string]any {
+	return supportmaps.Only(s.attributes, keys...)
+}
+
+func (s *Session) PreviousUrl() string {
+	return s.Get("_previous.url").(string)
+}
+
+func (s *Session) Pull(key string, def ...any) any {
+	return supportmaps.Pull(s.attributes, key, def...)
+}
+
+func (s *Session) Push(key string, value any) sessioncontract.Session {
+	arr := s.Get(key, make([]any, 0)).([]any)
```

Comment:
Will panic if the value isn't `[]any`.

Reply:
I will remove this function, not needed.